### ISSUE TITLE
Different responses with different id order

### ISSUE THREAD

**Issue by bastianjoel** [OpenSlides/openslides-autoupdate-service#752]
~~This was only tested on `feature/remove-template-fields` I am not sure if this also happens on current main.~~

With the following autoupdate requests I get different returns from the autoupdate. The only difference between these is the order of the ids array. I consider *Resp 2* the correct response. The corresponding request had the ids sorted ascending. *Resp 1* is missing data of the motion linked to the tag. 

Req 1: 
```json
[{"ids":[2,1,3],"collection":"tag","fields":{"id":null,"meeting_id":null,"name":null,"tagged_ids":{"type":"generic-relation-list","fields":{"content_object_id":{"type":"generic-relation","fields":{"content_object_id":{"type":"generic-relation","fields":{"id":null,"text":null,"title":null}},"id":null,"text":null,"title":null}},"description":null,"id":null,"meeting_id":{"type":"relation","collection":"meeting","fields":{"id":null,"name":null}},"name":null,"number":null,"sequential_number":null,"text":null,"title":null}}}}]
```

Req 2:
```json
[{"ids":[1,2,3],"collection":"tag","fields":{"id":null,"meeting_id":null,"name":null,"tagged_ids":{"type":"generic-relation-list","fields":{"content_object_id":{"type":"generic-relation","fields":{"content_object_id":{"type":"generic-relation","fields":{"id":null,"text":null,"title":null}},"id":null,"text":null,"title":null}},"description":null,"id":null,"meeting_id":{"type":"relation","collection":"meeting","fields":{"id":null,"name":null}},"name":null,"number":null,"sequential_number":null,"text":null,"title":null}}}}]
```

Resp 1:
```json
{"agenda_item/28/id":28,"agenda_item/28/meeting_id":1,"agenda_item/4/id":4,"agenda_item/4/meeting_id":1,"assignment/2/description":"\u003cp\u003eB-Ware\u003c/p\u003e","assignment/2/id":2,"assignment/2/meeting_id":1,"assignment/2/sequential_number":2,"assignment/2/title":"2. Wahl","meeting/1/id":1,"meeting/1/name":"OpenSlides Demo","motion/2/id":2,"motion/2/meeting_id":1,"motion/3/id":3,"motion/3/meeting_id":1,"tag/1/id":1,"tag/1/meeting_id":1,"tag/1/name":"Taga Taga","tag/1/tagged_ids":["motion/2"],"tag/2/id":2,"tag/2/meeting_id":1,"tag/2/name":"Tag2","tag/2/tagged_ids":["assignment/2","agenda_item/4","agenda_item/28"],"tag/3/id":3,"tag/3/meeting_id":1,"tag/3/name":"Tag3","tag/3/tagged_ids":["motion/2","motion/3"]}
```

Resp 2:
```json
{"agenda_item/28/id":28,"agenda_item/28/meeting_id":1,"agenda_item/4/id":4,"agenda_item/4/meeting_id":1,"assignment/2/id":2,"assignment/2/meeting_id":1,"assignment/2/sequential_number":2,"assignment/2/title":"2. Wahl","meeting/1/id":1,"meeting/1/name":"OpenSlides Demo","motion/2/id":2,"motion/2/meeting_id":1,"motion/2/number":"1 - 1","motion/2/sequential_number":2,"motion/2/text":"\u003cp\u003eFl\u0026ouml;mklawertaw\u003c/p\u003e","motion/2/title":"Änderungsantrag zu 1","motion/3/id":3,"motion/3/meeting_id":1,"tag/1/id":1,"tag/1/meeting_id":1,"tag/1/name":"Taga Taga","tag/1/tagged_ids":["motion/2"],"tag/2/id":2,"tag/2/meeting_id":1,"tag/2/name":"Tag2","tag/2/tagged_ids":["assignment/2","agenda_item/4","agenda_item/28"],"tag/3/id":3,"tag/3/meeting_id":1,"tag/3/name":"Tag3","tag/3/tagged_ids":["motion/2","motion/3"]}
```

**Comment by ostcar** [comment]
I can not reproduce this. Can you test this with the current main and tell me how to reproduce it?

**Comment by bastianjoel** [comment]
I can still reproduce this on current main. I am using a fresh dev setup started with `make run-dev` from the Openslides main repo. For the request I used the latest performance tool with command `openslides-performance -u admin -p admin request /system/autoupdate?single=1 -b '<req>'`.

**Comment by ostcar** [comment]
I found and fixed the bug. It was introduced, when I changed the key-representation. This was done in the same PR as the rtf.

The bug was very critical and I can not belief, that it was possible to use OpenSlides with it. It happens on generic-relation-list when you request different fields from different collections. For example `assignment/description` and `motion/number`. The old core removed the fields from the request, that do not exist.

For example, it first tries to fetch `assignment/number`. This is an invalid field, so it removes it from the request and does not request `motion/number` anymore. Therefore the order, in which the objects where processed did matter.

So the client did probably get a no data in a lot of cases. This could happen with the first data on the request or when the data changed. Maybe there are many other issues, where this bug was responsible.
